app/interactors/create_group.rb

```diff
@@ -1,2 +1,13 @@
 class CreateGroup < StandardInteraction
+  def validate_input
+    context.fail!(errors: "invalid input") unless context.group_params
```

Comment:
@Enriikke so now that I've done a few Interactors in this way (using `validate_input`, `execute`, `validate_output`) I'm sort of wondering if this might not be ideal. I was initially thinking that I'd want `validate_input` to do some more complex validation but in reality I end up just checking, in all of my Interactors, that the parameters are correct. 

Do you think we'd be better served by a simple `initialize` method in each Interactor that would then throw an `ArgumentError` if something wasn't provided? It seems like if I did that I wouldn't need to define a new error if the params aren't provided BUT it would throw that error right away, not giving my controller a chance to serialize/render the JSON response.